### PR TITLE
[WIP] Fix tx with depth less than root depth + DEPTH_DIFF still marked as missing

### PR DESCRIPTION
**Pull request by hasyimibhar**
Attempt to diagnose issue #230

### CODE REVIEW

File: graph.go [hunk 1]
```diff
@@ -448,7 +449,10 @@ func (g *Graph) Missing() []TransactionID {
 	g.RLock()
 	missing := make([]TransactionID, 0, len(g.missing))
 
-	for id := range g.missing {
+	for id, depth := range g.missing {
+		if g.rootDepth > conf.GetMaxDepthDiff()+depth {
```
Comment (a-urth) [inline graph.go]: but how is this possible? such transactions should be deleted from missing on root depth update

**Comment by rkeene** [comment]
Tested, there was no change with this.

File: graph.go (hunk 1 above)
Comment (rkeene) [reply graph.go]: It's not clear -- it seems to have not been effective as we still got the error identified in #230 with this branch

Comment (hasyimibhar) [reply graph.go]: I am not sure yet. This is not an actual fix, just trying to see if doing this makes the error go away. Looks like the error is still there.

**Comment by rkeene** [comment]
Let's try something else ! 👍

**Pull request closed without merging**